modules/apps/oauth2-provider/oauth2-provider-web/src/main/java/com/liferay/oauth2/provider/web/internal/portlet/action/UpdateOAuth2ApplicationMVCActionCommand.java

```diff
@@ -158,7 +160,8 @@ public boolean processAction(
 						allowedGrantTypesList, clientCredentialUserId, clientId,
 						clientProfile.id(), clientSecret, description,
 						featuresList, homePageURL, 0, name, privacyPolicyURL,
-						redirectURIsList, scopeAliasesList, serviceContext);
```

Comment:
Hey @martamedio . I think ServiceContext must always be the last parameter when it is present.

Reply:
Thanks for the heads Up

@csierra I could be a good check for our bot

Reply:
I am not against this... but this is something that maybe would belong more to the realms of source formatter...

actually I am kind of surprised it does not do this already... I will chek